nautobot/ipam/api/serializers.py

```diff
@@ -184,6 +184,14 @@ class Meta:
 class PrefixSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
     prefix = IPFieldSerializer()
     type = ChoiceField(choices=PrefixTypeChoices, default=PrefixTypeChoices.TYPE_NETWORK)
+    # for backward compatibility with 2.0-2.1 where a Prefix had only a single Location
+    location = NautobotHyperlinkedRelatedField(
```

Comment:
Why should users be able to utilise a `pre-2.2` feature in `2.2`? I had assumed that these features were reserved for legacy versions.

Reply:
Fair question. #5052 did the same thing with a `location` write-only field on the 2.2 `VLANSerializer` - I think the primary motivation was to allow CSV import to still specify a location if desired. (https://github.com/nautobot/nautobot/pull/5052#discussion_r1459014038)